File: nv.py
import requests

url = "https://nvspc.biz/api/u4H-5t3hM0eMprf6kowrpw/getShowcase"

#  querystring = {"c":""}

#  headers = {'user-agent': 'vscode-restclient'}

r = requests.post(url)
# response = requests.get(url)

# r.raise_for_status()  # raises exception when not a 2xx response
# if r.status_code != 204:
# r.text

print(r.text)

File: nv.js
const https = require('https');

// const options = {
  // host: 'nvzzz.ru/api/dummy',
  // path: '/getShowcase',
  // method: 'GET'
// };

// let request = https.request(options, (res) => { });

// const https = require('https');



let request = https.get('https://nvspc.biz/api/u4H-5t3hM0eMprf6kowrpw', (res) => {
  if (res.statusCode !== 200) {
    console.error(`Did not get an OK from the server. Code: ${res.statusCode}`);
    res.resume();
    return;
  }

  let data = '';

  res.on('data', (chunk) => {
    data += chunk;
  });

  res.on('close', () => {
    console.log('Retrieved all data');
    console.log(JSON.parse(data));
  });
});
